main.cpp

```diff
@@ -103,12 +104,16 @@ void someMemberFunction(const Axe& axe);
         float preheatOven(float ovenTemperature);
         bool tableSet(int numPlacesToSet, int placesSet);
         void printNumKnives();
+
+        JUCE_LEAK_DETECTOR(Kitchen)
     };
-    bool furnishHouse(House myHouse);
+    bool furnishHouse(const House& myHouse);
     void openFrontDoor(std::string door = "front door");
-    bool closeWindow(House::Kitchen myKitchen);
+    bool closeWindow(House::Kitchen& myKitchen);
```

Comment:
pass by `const reference` if you're not modifying the passed-in object
- [ ] revise